test/preflight/fly_launch_test.go

```diff
@@ -43,6 +43,7 @@ func TestFlyLaunch_case01(t *testing.T) {
 			"auto_stop_machines":   true,
 			"auto_start_machines":  true,
 			"min_machines_running": int64(0),
+			"processes":            append(make([]interface{}, 0), "app"),
```

Comment:
that reads so weird :))   use `[]any{"app"}` instead 🙏

Reply:
Ah, I'd assumed this didn't work but just realised I tried `[]interface{"app"}` instead of `[]interface{}{"app"}`. Great suggestion, added it.